configure.ac

```diff
@@ -5,13 +5,10 @@ AC_CONFIG_HEADERS([config.h])
 AC_PROG_CC
 AM_PROG_CC_C_O
 AC_CHECK_FUNCS([pledge reallocarray strtonum])
-AC_SEARCH_LIBS([setupterm], [curses], [],
-  [
-    AC_SEARCH_LIBS([setupterm], [ncursesw],
-      [AC_DEFINE([HAVE_NCURSESW_H], [1], [Define if ncursesw is available])],
-      [AC_MSG_ERROR([unable to find setupterm function])]
-    )
-  ]
-)
+PKG_CHECK_MODULES([NCURSES], [ncursesw],
+  [AC_DEFINE([HAVE_NCURSESW_H], [1], [Define if ncursesw is available])],
+  [PKG_CHECK_MODULES([NCURSES], [ncurses])])
+CFLAGS="$CFLAGS $NCURSES_CFLAGS"
+LIBS="$LIBS $NCURSES_LIBS"
```

Comment:
These should probably modify `AM_CPPFLAGS` and `AM_LDFLAGS` in the `Makefile.am`, like so:

```Makefile
AM_CPPFLAGS = $(NCURSES_CFLAGS)
AM_LDFLAGS = $(NCURSES_LIBS)
```

This way the end user (the person compiling from the tarball) can override `CFLAGS` from the `make` command line.

(I'm not 100% sure I understand the original problem being solved, so let me know if I have something obvious wrong.)

Reply:
My bad, here's the updated [commit](https://github.com/calleerlandsson/pick/commit/8c79e57a2e62a242aa9e3b53684607282afb8076) to review (should have made that clearer).

Reply:
Aha! LGTM. One optional suggestion.